app/views/comments/edit.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= form_for @comment do |c| %>
+  <%= c.label :title %>
+  <%= c.text_field :title %><br>
```

Comment:
`<br>` tags are usually not the best way to apply style. I'd suggest wrapping each pair of label and input in it's own div.